extensions/godslayerakp/http.js

```diff
@@ -0,0 +1,445 @@
+(function(Scratch) {
+    'use strict';
+    if (!Scratch.extensions.unsandboxed) throw 'can not load out side unsandboxed mode';
+
+    const {vm} = Scratch;
+    const {runtime, extensionManager} = vm;
+
+    // inject comments bc yes
+    // @ts-ignore
+    const oldConvert = runtime._convertForScratchBlocks.bind(runtime);
+    const createComment = blockInfo => ({
+        info: blockInfo,
+        xml: `<label text="${blockInfo.slice(3)}"></label>`
+    });
+    // @ts-ignore
+    runtime._convertForScratchBlocks = (blockInfo, categoryInfo) => {

```

Comment:
use

blockType: Scratch.BlockType.LABEL,
text: "text"

instead of that insecure mess

Reply:
its not insecure? and this merge was pre-labels but yes it needs changed

Reply:
fixed...